integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/SparkCommonIT.java

```diff
@@ -674,9 +674,19 @@ protected String getExpectedTableData(SparkTableInfo table) {
   }
 
   protected String getCreateSimpleTableString(String tableName) {
-    return String.format(
-        "CREATE TABLE %s (id INT COMMENT 'id comment', name STRING COMMENT '', age INT)",
-        tableName);
+    return getCreateSimpleTableString(tableName, false);
+  }
+
+  protected String getCreateSimpleTableString(String tableName, boolean isExternal) {
```

Comment:
following code seems more extensible, we may add more parameters beside isExternal.
```java
String external = "";
if (isExternal) {
  external = "EXTERNAL";
}

String.format(
          "CREATE %s TABLE %s (id INT COMMENT 'id comment', name STRING COMMENT '', age INT)",
          external,tableName);

```

Reply:
@charliecheng630 , could you fix this?

Reply:
Updated.